scripts/data/src/org/akvo/gae/remoteapi/EnsureHouseholdPrivacy.java

```diff
@@ -0,0 +1,46 @@
+package org.akvo.gae.remoteapi;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.Filter;
+import com.google.appengine.api.datastore.Query.FilterOperator;
+import com.google.appengine.api.datastore.Query.FilterPredicate;;
+
+public class EnsureHouseholdPrivacy implements Process {
+
+	@Override
+	public void execute(DatastoreService ds, String[] args) throws Exception {
+		
+	    if (args.length != 0) {
+            System.err.println("Usage: " + RemoteAPI.class.getName()
+                    + "EnsureHouseholdPrivacy <appid> <username> <password>");
+            System.exit(1);
+        }
+
+        Query surveyQuery = new Query("SurveyGroup");
```

Comment:
This query could already include a filter for `SurveyGroup where projectType = 'PROJECT'`